platform-controller/src/test/java/org/hobbit/controller/docker/ClusterManagerImplTest.java

```diff
@@ -15,11 +16,11 @@ public void setUp() throws Exception {
         clusterManager = new ClusterManagerImpl();
     }
 
-    @Test
-    public void getClusterInfo() throws Exception {
-        final Info info = clusterManager.getClusterInfo();
-        assertNotNull(info);
-    }
+//    @Test
```

Comment:
This is not a really good solution. There are several options:
1. Make this test conditional based on the environment, i.e., whether it is in an environment in which the test can run or not.
2. Add a `@Ignore` to the class, if you want it to be ignored.